lib/kubernetes-deploy/resource_watcher.rb

```diff
@@ -41,14 +41,23 @@ def run(delay_sync: 3.seconds, reminder_interval: 30.seconds)
     def report_what_just_happened(resources, watch_time)
       new_successes, new_failures = resources.partition(&:deploy_succeeded?)
       new_failures.each do |resource|
+        tags = %W(namespace:#{@namespace} sha:#{ENV['REVISION']} resource:#{resource.id})
         if resource.deploy_failed?
+          ::StatsD.increment('deploy.resource.status', 1, tags: tags << "status:failed")
```

Comment:
Sorry, I just realized we're reporting this from the watcher, which is used by the other task types (restart and run) as well. Maybe the metric should just be `resource.status` then... wdyt @n1koo @kirs , if you're looking at that metric, is it a good or a confusing thing that you'd see data from how quickly it started during a restart task as well?